spec/factorial_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../solver'
+
+RSpec.describe 'factorial method of the solver class' do
+  def factorial(number)
+    factor = Solver.new
+    factor.factorial(number)
+  end
+  it 'should be able to calculate the factorial of 5' do
+    expect(factorial(5)).to eq(120)
+  end
+  it 'should be able to calculate the factorial of 0' do
+    expect(factorial(0)).to eq(1)
+  end
```

Comment:
- [ ] Nice job here testing the factorial method, As per this [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/testing/tdd_project.md#:~:text=positive%20integers%2C%20so-,if%20a%20negative%20integer%20is%20given%20it%20should%20raise%20an%20exception.,-Create%20a%20method), Please add a test to raise an exception when you try to get the factorial of a negative number to cover all edge cases.